rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
> So in this case, when looking for event7's modalias, it doesn't visit it's immediate parent (input8) but instead goes straight to the grandfather ACCE0001:00 which was selected by SUBSYSTEMS. That's why the rule in my last comment works.

I don't understand why the modalias it tries to match on isn't the ACPI device's, and I don't have any more idea on how to fix this, other than experimenting. One thing that's certain though, we don't want the specific driver name for that accelerometer in the matching rules, otherwise we'd need to add matching rules for each and every "input" accelerometer driver.